scripts/stagingaddrs/main.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
maybe add a comment to explain why we need this: `// Command stagtingaddrs prints out .. because ...`